### PR TITLE
Rename UOR to Emporous

### PR DESCRIPTION
**Pull request by d10n**
Merge after #1

### CODE REVIEW

**Comment by d10n** [comment]
Rebased on main

File: manager.proto [hunk 1]
```diff
@@ -76,7 +76,7 @@ message File {
 message AuthConfig {
   string username = 1;
   string password = 2;
-  string registry_host = 3;
+  string server_address = 3;
```
Comment (jpower432) [inline manager.proto]: Looks like this got changed back to `server_address`

**Review by jpower432 — COMMENTED** [review]
Looks good. One issue I found.

File: sample_client.py [hunk 2]
```diff
@@ -52,7 +55,7 @@ def create_auth_config(reference: str) -> pb2.AuthConfig:
     creds = auth.resolve_authconfig(auth_configs, registry)
     if creds is not None:
         auth_config = pb2.AuthConfig(
-            registry_host=registry,
+            server_address=registry,
```
Comment (jpower432) [inline sample_client.py]: Found one more reference to server address

**Review by jpower432 — APPROVED** [review]
LGTM

**Pull request merged**